meal-planner-fredericton-ui/src/index.jsx

```diff
@@ -31,13 +31,14 @@ const theme = createMuiTheme({
       fontFamily: 'Open Sans',
     },
     text: {
-      primary: '#ffffff',
+      primary: '#000000',
```

Comment:
Delete Dialog title was not visible as it was in white. So fixed it to black to be shown.